index.html

```diff
@@ -9,26 +9,28 @@
     <script src="https://cdn.tailwindcss.com"></script>
     <title>Beer and a Show</title>
     <link rel="stylesheet" href="./assets/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Bruno+Ace&family=Bruno+Ace+SC&display=swap" rel="stylesheet">
 </head>
 
 
 <body class="font-family: arial black">
-    <header class="p-20 text-lg justify-center bg-gradient-to-r from-cyan-900 via-cyan-700 to-sky-900 h14">
+    <header class="p-20 text-lg justify-center bg-gradient-to-r from-blue-950 via-teal-600 to-blue-950 h14">
         <img src="" alt="">
-        <h1 class="title text-6xl font-bold">Beer and a Show</h1>
+        <h1 class="title text-10xl font-bold">Beer and a Show</h1>
     </header>
-    <div id="quote" class="p-10 underline underline-offset-8 text-3xl font-bold">Heading on a trip? Enter the city you
-        are travelling to for a list of 6 local breweries and events coming up! Use the drop down to filter for just
-        Events or Breweries.</div>
+    <div id="quote" class="p-5 underline-offset-8 text-4xl font-bold">Plan your whole night out in one convenient place!</div>
+        <div id="quote" class="p-5 underline-offset-8 text-2xl font-bold">Enter any city to search for a list of 6 local breweries and 6 upcoming events!</div>
```

Comment:
'.....for a list of 6 local breweries and 6 upcoming events'- repeated, so better to remove either of the two